tests/logging_tests/tests.py

```diff
@@ -105,6 +106,47 @@ def test_django_logger_debug(self):
             self.logger.debug('debug')
             self.assertEqual(self.logger_output.getvalue(), '')
 
+    def test_page_found_no_warning(self):
+        with self.settings(DEBUG=True):
+            self.client.get('/innocent/')
+            self.assertEqual(self.logger_output.getvalue(), '')
+
+    def test_page_not_found_warning(self):
+        with self.settings(DEBUG=True):
+            self.client.get('/does_not_exist/')
+            self.assertEqual(self.logger_output.getvalue(), 'Not Found: /does_not_exist/\n')
+
+
+@override_settings(
+    USE_I18N=True,
+    LANGUAGES=[
+        ('en', 'English'),
+    ],
+    MIDDLEWARE_CLASSES=[
+        'django.middleware.locale.LocaleMiddleware',
+        'django.middleware.common.CommonMiddleware',
+    ],
+    ROOT_URLCONF='logging_tests.urls_i18n'
```

Comment:
Add a trailing comma so if more items are added later we don't have to modify this line again.